node/10-user/Dockerfile

```diff
@@ -14,6 +14,14 @@
 
 FROM node:10-stretch
 
+# Add Graphviz
+RUN set -ex; \
+  apt-get update -y; \
+  apt-get install -y \
+    graphviz \
+    ; \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
Why do we need to delete the package information?

Reply:
It is standard practice in Dockerfiles to clean up package information/caches related to packager tooling to create lean images. This is the standard approach recommended with apt-get.